MajanScoreManager/Views/Winning.swift

```diff
@@ -57,22 +58,27 @@ struct Winning: View {
                     }
                 }
                 .frame(height: CGFloat(2) * (Winning.rowHeight + Winning.rowMargin))
-
-                Text("和了者と放銃者を選んでください")
+                //和了の種類選択後、和了者を選択可能にする
+                if type != WinningType.unselected{
+                Text("プレイヤーを選択してください")
```

Comment:
```suggestion
                    Text("プレイヤーを選択してください")
```
L63からL78まで`if`文の中なんでインデントおなしゃす